rasterio/rpc.py

```diff
@@ -0,0 +1,68 @@
+from rasterio.compat import Iterable, UserDict, text_type
```

Comment:
@underchemist version 1.2.0 won't support Python 2, so we shouldn't import from the compat module.